armi/reactor/tests/test_components.py

```diff
@@ -1149,6 +1280,13 @@ def test_getCircleInnerDiameter(self):
 
 
 class TestHelix(TestShapedComponent):
+    """Test helix shaped component.
+
+    .. test:: Helix shaped component
+        :id: T_ARMI_COMP_SHAPES7
+        :tests: R_ARMI_COMP_SHAPES
+    """
+
```

Comment:
Again, delete this.  Or move it down to one-or-more of the tests in this class.

We can't have class-level test crumbs. Not your fault, you didn't know.